_posts/blog/2019-10-03-2019-10-03-lab-meeting.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: Lab Meeting on 3 October 2019 ### REPLACE `DATE` with the date, eg: 18 July 2019
+categories: blog
+excerpt: Notes from the group's weekly lab meeting ### ADD a short description (or keep that one if you'd)
+tags: [lab-meeting] ### INSERT TAGS IF APPLICABLE
+image:
+  feature:
+link:
+date: 2019-10-03 ### UPDATE date
+modified:
+share: true
+author: kirstie_whitaker ### CHANGE to your author name (in _data/authors.yml)
+---
+
+There was no lab meeting this week because there was a Turing Town Hall (where we get to grill the [Institute Director](https://www.turing.ac.uk/people/leadership/adrian-smith)).
+
+We are delighted to share this video from May of the [Pint of Science](https://www.turing.ac.uk/events/pint-science) panel discussion featuring Kirstie Whitaker, [Dianna Smith](https://twitter.com/geodianna?lang=en), [Rebecca Sterry](https://twitter.com/bex_sterry), [Suzanne Iwai](https://twitter.com/SuzanneIwai) and Georgia Aitkenhead on **Empowering autistic people through citizen science**.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/DNisoU9uXqk" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
This video doesn't render in markdown (but it might not need to?)

Reply:
Thanks @sgibson91 - the video _does_ render in jekyll but I appreciate you checking ✨